addons/io/org.openhab.io.hueemulation/README.md

```diff
@@ -10,7 +10,7 @@ Hue Emulation exposes openHAB items as Hue devices to other Hue HTTP API compati
 * Pairing (security) can be enabled/disabled in real time using the configuration service (under services in the PaperUI for example)  
 
 ## Configuration:
-
+in "userdata/config/org/openhab/hueemulation.config"
```

Comment:
this was only a suggestion to mention where the config file is located. Did not find it at once.